governance/SDK.md

```diff
@@ -0,0 +1,172 @@
+
+We define a project `cloudevents/sdk-x` _not actively maintained_ if:
+
+6. No commits on master from `sdk-x-maintainers` since 4 months
+7. Issues and/or PRs are not being triaged from `sdk-x-mantainers` since 2
+   months
+8. Security patches are not being **released** from `sdk-x-mantainers` since 1
+   months from CVE disclosures
+
+To prevent projects _not actively maintained_, we define different actions the
+community might take:
+
+- Temporary security patches delivered by an _sdk maintainer_ not part of
+  `sdk-x-maintainers` group
+- Handover of the project to a new maintainer
+- Archive the project
+
+### Security patches
+
+If a project `cloudevents/sdk-x` meets _healthy_ criteria 1, 2, 4, but the
+`sdk-x-maintainers` group is not actively performing security patches as defined
+in criteria 8, the community could decide to entitle one or more members from
+the larger _sdk maintainers_ group to perform security patches and release new
+minor versions of the project.
+
+Because the people entitled to perform security patches needs write rights on
+the repo, the community MUST temporary assign him/them the rights to perform it.
+After the release of the patch, these rights MUST be removed from the member(s)
+who took care of it.
+
+Because of the urgency nature of security patches, an _informal_ vote during the
+CloudEvents SDK meeting is enough to assign the security patch to the _sdk
+maintainer(s)_ which volunteers to take care of it.
+
+NOTE: this process is valid only for security patches, not for bugs or
+enhancements.
+
+### Handover to a new maintainer/group of maintainers
+
+It may happen that, if a project is not following the criteria 1, 2, 4, 5, 6, 7,
+8, the community might decide to handover the project to a new maintainer/group
+of maintainers. The community can perform the handover to a new maintainer
+**iff** a voting process to add such maintainer was **already performed** with
+100% of entitled people abstained and:
+
+- Or the new maintainer(s) already meet the requirements enlisted in
+  [New maintainers section](#new-maintainers),
+- Or, because of the inactivity of the project due to criteria 7, the new
+  maintainer(s) performed the required PRs/Reviews/Code changes to meet the
+  [New maintainers criteria](#new-maintainers) but he/they wasn't/weren't able
+  to merge them.
+
+The new maintainer must show the good will to keep the project healthy and
+community could ask for an eventual plan to improve the quality of the project.
+
+As soon as the new maintainer is identified, the community can proceed to vote
+to handover the project using the
+[Asynchronous voting process](#asynchronous-voting-process). If more than one
+new maintainer wants to handover the project, separate voting process must be
+done for all of them, and both of them must follow the same voting criteria (and
+not the ones from [New maintainers section](#new-maintainers)).
+
+The voting criteria are:
```

Comment:
What happens if the vote is "no"?

Reply:
It's explained below. If no the project remains as is. The community could decide to take the path of archiving the project